/bin/bash /home/jinyao/PycharmProjects/IADBE/train_test_btech_ganomaly.sh
(IADBE) jinyao@jinyao-System-Product-Name:~/PycharmProjects/IADBE$ /bin/bash /home/jinyao/PycharmProjects/IADBE/train_test_btech_ganomaly.sh
Running command: anomalib train --data anomalib.data.BTech --data.category 01 --config ./configs/models/ganomaly.yaml
2024-06-25 16:36:54,061 - anomalib.utils.config - WARNING - Anomalib currently does not support multi-gpu training. Setting devices to 1.
[06/25/24 16:36:54] WARNING  Anomalib currently does not support multi-gpu training. Setting devices to 1.                                                                                                    config.py:262
2024-06-25 16:36:54,078 - anomalib.models.components.base.anomaly_module - INFO - Initializing Ganomaly model.
                    INFO     Initializing Ganomaly model.                                                                                                                                              anomaly_module.py:42
2024-06-25 16:36:54,084 - anomalib.callbacks - INFO - Loading the callbacks
                    INFO     Loading the callbacks                                                                                                                                                           __init__.py:43
2024-06-25 16:36:54,093 - anomalib.engine.engine - INFO - Overriding gradient_clip_val from None with 0 for Ganomaly
                    INFO     Overriding gradient_clip_val from None with 0 for Ganomaly                                                                                                                        engine.py:84
2024-06-25 16:36:54,094 - anomalib.engine.engine - INFO - Overriding num_sanity_val_steps from None with 0 for Ganomaly
                    INFO     Overriding num_sanity_val_steps from None with 0 for Ganomaly                                                                                                                     engine.py:84
2024-06-25 16:36:54,173 - lightning.pytorch.utilities.rank_zero - INFO - Trainer already configured with model summary callbacks: [<class 'lightning.pytorch.callbacks.rich_model_summary.RichModelSummary'>]. Skipping setting a default `ModelSummary` callback.
[06/25/24 16:36:54] INFO     Trainer already configured with model summary callbacks: [<class 'lightning.pytorch.callbacks.rich_model_summary.RichModelSummary'>]. Skipping setting a default               rank_zero.py:63
                             `ModelSummary` callback.
2024-06-25 16:36:54,181 - lightning.pytorch.utilities.rank_zero - INFO - GPU available: True (cuda), used: True
                    INFO     GPU available: True (cuda), used: True                                                                                                                                         rank_zero.py:63
2024-06-25 16:36:54,182 - lightning.pytorch.utilities.rank_zero - INFO - TPU available: False, using: 0 TPU cores
                    INFO     TPU available: False, using: 0 TPU cores                                                                                                                                       rank_zero.py:63
2024-06-25 16:36:54,183 - lightning.pytorch.utilities.rank_zero - INFO - IPU available: False, using: 0 IPUs
                    INFO     IPU available: False, using: 0 IPUs                                                                                                                                            rank_zero.py:63
2024-06-25 16:36:54,183 - lightning.pytorch.utilities.rank_zero - INFO - HPU available: False, using: 0 HPUs
                    INFO     HPU available: False, using: 0 HPUs                                                                                                                                            rank_zero.py:63
2024-06-25 16:36:54,184 - anomalib.models.components.base.anomaly_module - WARNING - No implementation of `configure_transforms` was provided in the Lightning model. Using default transforms from the base class. This may not be suitable for your use case. Please override `configure_transforms` in your model.
                    WARNING  No implementation of `configure_transforms` was provided in the Lightning model. Using default transforms from the base class. This may not be suitable for your use     anomaly_module.py:235
                             case. Please override `configure_transforms` in your model.
2024-06-25 16:36:54,185 - anomalib.data.image.btech - INFO - Found the dataset.
                    INFO     Found the dataset.                                                                                                                                                                btech.py:351
2024-06-25 16:36:54,205 - lightning.pytorch.utilities.rank_zero - INFO - You are using a CUDA device ('NVIDIA GeForce RTX 3090') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' | 'high')` which will trade-off precision for performance. For more details, read https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
                    INFO     You are using a CUDA device ('NVIDIA GeForce RTX 3090') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' |         rank_zero.py:63
                             'high')` which will trade-off precision for performance. For more details, read
                             https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
2024-06-25 16:36:55,378 - lightning.pytorch.accelerators.cuda - INFO - LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
[06/25/24 16:36:55] INFO     LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]                                                                                                                                           cuda.py:58
┏━━━┳━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┓
┃   ┃ Name                  ┃ Type                     ┃ Params ┃
┡━━━╇━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━┩
│ 0 │ generator_loss        │ GeneratorLoss            │      0 │
│ 1 │ discriminator_loss    │ DiscriminatorLoss        │      0 │
│ 2 │ _transform            │ Compose                  │      0 │
│ 3 │ normalization_metrics │ MinMax                   │      0 │
│ 4 │ image_threshold       │ F1AdaptiveThreshold      │      0 │
│ 5 │ pixel_threshold       │ F1AdaptiveThreshold      │      0 │
│ 6 │ image_metrics         │ AnomalibMetricCollection │      0 │
│ 7 │ pixel_metrics         │ AnomalibMetricCollection │      0 │
│ 8 │ model                 │ GanomalyModel            │  188 M │
└───┴───────────────────────┴──────────────────────────┴────────┘
Trainable params: 188 M
Non-trainable params: 0
Total params: 188 M
Total estimated model params size (MB): 754
/home/jinyao/anaconda3/envs/IADBE/lib/python3.10/site-packages/lightning/pytorch/core/module.py:494: You called `self.log('generator_loss', ..., logger=True)` but have no logger configured. You can enable one by doing
`Trainer(logger=ALogger(...))`
/home/jinyao/anaconda3/envs/IADBE/lib/python3.10/site-packages/lightning/pytorch/core/module.py:494: You called `self.log('discriminator_loss', ..., logger=True)` but have no logger configured. You can enable one by
doing `Trainer(logger=ALogger(...))`
Epoch 6/99 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 13/13 0:00:07 • 0:00:00 3.33it/s generator_loss_step: 42.880 discriminator_loss_step: 23.839 image_AUROC: 0.794 image_PRO: 1.000 generator_loss_epoch: 44.414
                                                                                     discriminator_loss_epoch: 23.871
[06/25/24 16:38:37] INFO     Training took 102.50 seconds                                                                                                                                                                timer.py:59
Epoch 6/99 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 13/13 0:00:07 • 0:00:00 3.33it/s generator_loss_step: 42.880 discriminator_loss_step: 23.839 image_AUROC: 0.732 image_PRO: 1.000 generator_loss_epoch: 42.912
                                                                                     discriminator_loss_epoch: 23.871
2024-06-25 16:38:37,897 - anomalib.data.image.btech - INFO - Found the dataset.
                    INFO     Found the dataset.                                                                                                                                                                         btech.py:351
2024-06-25 16:38:38,155 - lightning.pytorch.accelerators.cuda - INFO - LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
[06/25/24 16:38:38] INFO     LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]                                                                                                                                                    cuda.py:58
Testing ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3/3 0:00:07 • 0:00:00 0.26it/s  2024-06-25 16:38:49,658 - anomalib.callbacks.timer - INFO - Testing took 10.723740816116333 seconds
Throughput (batch_size=32) : 6.527572905790437 FPS
[06/25/24 16:38:49] INFO     Testing took 10.723740816116333 seconds                                                                                                                                                    timer.py:109
                             Throughput (batch_size=32) : 6.527572905790437 FPS
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃        Test metric        ┃       DataLoader 0        ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│        image_AUROC        │    0.7317784428596497     │
│         image_PRO         │            1.0            │
└───────────────────────────┴───────────────────────────┘
Testing ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3/3 0:00:07 • 0:00:00 0.26it/s
Running command: anomalib train --data anomalib.data.BTech --data.category 02 --config ./configs/models/ganomaly.yaml
2024-06-25 16:38:56,161 - anomalib.utils.config - WARNING - Anomalib currently does not support multi-gpu training. Setting devices to 1.
[06/25/24 16:38:56] WARNING  Anomalib currently does not support multi-gpu training. Setting devices to 1.                                                                                                             config.py:262
2024-06-25 16:38:56,178 - anomalib.models.components.base.anomaly_module - INFO - Initializing Ganomaly model.
                    INFO     Initializing Ganomaly model.                                                                                                                                                       anomaly_module.py:42
2024-06-25 16:38:56,184 - anomalib.callbacks - INFO - Loading the callbacks
                    INFO     Loading the callbacks                                                                                                                                                                    __init__.py:43
2024-06-25 16:38:56,185 - anomalib.engine.engine - INFO - Overriding gradient_clip_val from None with 0 for Ganomaly
                    INFO     Overriding gradient_clip_val from None with 0 for Ganomaly                                                                                                                                 engine.py:84
2024-06-25 16:38:56,186 - anomalib.engine.engine - INFO - Overriding num_sanity_val_steps from None with 0 for Ganomaly
                    INFO     Overriding num_sanity_val_steps from None with 0 for Ganomaly                                                                                                                              engine.py:84
2024-06-25 16:38:56,266 - lightning.pytorch.utilities.rank_zero - INFO - Trainer already configured with model summary callbacks: [<class 'lightning.pytorch.callbacks.rich_model_summary.RichModelSummary'>]. Skipping setting a default `ModelSummary` callback.
[06/25/24 16:38:56] INFO     Trainer already configured with model summary callbacks: [<class 'lightning.pytorch.callbacks.rich_model_summary.RichModelSummary'>]. Skipping setting a default `ModelSummary`         rank_zero.py:63
                             callback.
2024-06-25 16:38:56,274 - lightning.pytorch.utilities.rank_zero - INFO - GPU available: True (cuda), used: True
                    INFO     GPU available: True (cuda), used: True                                                                                                                                                  rank_zero.py:63
2024-06-25 16:38:56,274 - lightning.pytorch.utilities.rank_zero - INFO - TPU available: False, using: 0 TPU cores
                    INFO     TPU available: False, using: 0 TPU cores                                                                                                                                                rank_zero.py:63
2024-06-25 16:38:56,275 - lightning.pytorch.utilities.rank_zero - INFO - IPU available: False, using: 0 IPUs
                    INFO     IPU available: False, using: 0 IPUs                                                                                                                                                     rank_zero.py:63
2024-06-25 16:38:56,276 - lightning.pytorch.utilities.rank_zero - INFO - HPU available: False, using: 0 HPUs
                    INFO     HPU available: False, using: 0 HPUs                                                                                                                                                     rank_zero.py:63
2024-06-25 16:38:56,277 - anomalib.models.components.base.anomaly_module - WARNING - No implementation of `configure_transforms` was provided in the Lightning model. Using default transforms from the base class. This may not be suitable for your use case. Please override `configure_transforms` in your model.
                    WARNING  No implementation of `configure_transforms` was provided in the Lightning model. Using default transforms from the base class. This may not be suitable for your use case. Please anomaly_module.py:235
                             override `configure_transforms` in your model.
2024-06-25 16:38:56,278 - anomalib.data.image.btech - INFO - Found the dataset.
                    INFO     Found the dataset.                                                                                                                                                                         btech.py:351
2024-06-25 16:38:56,297 - lightning.pytorch.utilities.rank_zero - INFO - You are using a CUDA device ('NVIDIA GeForce RTX 3090') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' | 'high')` which will trade-off precision for performance. For more details, read https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
                    INFO     You are using a CUDA device ('NVIDIA GeForce RTX 3090') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' | 'high')` which   rank_zero.py:63
                             will trade-off precision for performance. For more details, read https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
2024-06-25 16:38:57,469 - lightning.pytorch.accelerators.cuda - INFO - LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
[06/25/24 16:38:57] INFO     LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]                                                                                                                                                    cuda.py:58
┏━━━┳━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┓
┃   ┃ Name                  ┃ Type                     ┃ Params ┃
┡━━━╇━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━┩
│ 0 │ generator_loss        │ GeneratorLoss            │      0 │
│ 1 │ discriminator_loss    │ DiscriminatorLoss        │      0 │
│ 2 │ _transform            │ Compose                  │      0 │
│ 3 │ normalization_metrics │ MinMax                   │      0 │
│ 4 │ image_threshold       │ F1AdaptiveThreshold      │      0 │
│ 5 │ pixel_threshold       │ F1AdaptiveThreshold      │      0 │
│ 6 │ image_metrics         │ AnomalibMetricCollection │      0 │
│ 7 │ pixel_metrics         │ AnomalibMetricCollection │      0 │
│ 8 │ model                 │ GanomalyModel            │  188 M │
└───┴───────────────────────┴──────────────────────────┴────────┘
Trainable params: 188 M
Non-trainable params: 0
Total params: 188 M
Total estimated model params size (MB): 754
/home/jinyao/anaconda3/envs/IADBE/lib/python3.10/site-packages/lightning/pytorch/core/module.py:494: You called `self.log('generator_loss', ..., logger=True)` but have no logger configured. You can enable one by doing
`Trainer(logger=ALogger(...))`
/home/jinyao/anaconda3/envs/IADBE/lib/python3.10/site-packages/lightning/pytorch/core/module.py:494: You called `self.log('discriminator_loss', ..., logger=True)` but have no logger configured. You can enable one by doing
`Trainer(logger=ALogger(...))`
Epoch 3/99 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 13/13 0:00:03 • 0:00:00 4.97it/s generator_loss_step: 16.954 discriminator_loss_step: 0.000 image_AUROC: 0.309 image_PRO: 1.000 generator_loss_epoch: 18.869
                                                                                     discriminator_loss_epoch: 0.000
[06/25/24 16:39:31] INFO     Training took 34.04 seconds                                                                                                                                                                 timer.py:59
Epoch 3/99 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 13/13 0:00:03 • 0:00:00 4.97it/s generator_loss_step: 16.954 discriminator_loss_step: 0.000 image_AUROC: 0.321 image_PRO: 1.000 generator_loss_epoch: 17.434
                                                                                     discriminator_loss_epoch: 0.000
2024-06-25 16:39:31,524 - anomalib.data.image.btech - INFO - Found the dataset.
                    INFO     Found the dataset.                                                                                                                                                                         btech.py:351
2024-06-25 16:39:31,689 - lightning.pytorch.accelerators.cuda - INFO - LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
[06/25/24 16:39:31] INFO     LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]                                                                                                                                                    cuda.py:58
Testing ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 8/8 0:00:12 • 0:00:00 0.57it/s  2024-06-25 16:39:45,188 - anomalib.callbacks.timer - INFO - Testing took 13.011890888214111 seconds
Throughput (batch_size=32) : 17.676139615367433 FPS
[06/25/24 16:39:45] INFO     Testing took 13.011890888214111 seconds                                                                                                                                                    timer.py:109
                             Throughput (batch_size=32) : 17.676139615367433 FPS
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃        Test metric        ┃       DataLoader 0        ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│        image_AUROC        │    0.3499999940395355     │
│         image_PRO         │            1.0            │
└───────────────────────────┴───────────────────────────┘
Testing ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 8/8 0:00:12 • 0:00:00 0.57it/s
Running command: anomalib train --data anomalib.data.BTech --data.category 03 --config ./configs/models/ganomaly.yaml
2024-06-25 16:39:51,360 - anomalib.utils.config - WARNING - Anomalib currently does not support multi-gpu training. Setting devices to 1.
[06/25/24 16:39:51] WARNING  Anomalib currently does not support multi-gpu training. Setting devices to 1.                                                                                                             config.py:262
2024-06-25 16:39:51,377 - anomalib.models.components.base.anomaly_module - INFO - Initializing Ganomaly model.
                    INFO     Initializing Ganomaly model.                                                                                                                                                       anomaly_module.py:42
2024-06-25 16:39:51,383 - anomalib.callbacks - INFO - Loading the callbacks
                    INFO     Loading the callbacks                                                                                                                                                                    __init__.py:43
2024-06-25 16:39:51,384 - anomalib.engine.engine - INFO - Overriding gradient_clip_val from None with 0 for Ganomaly
                    INFO     Overriding gradient_clip_val from None with 0 for Ganomaly                                                                                                                                 engine.py:84
2024-06-25 16:39:51,385 - anomalib.engine.engine - INFO - Overriding num_sanity_val_steps from None with 0 for Ganomaly
                    INFO     Overriding num_sanity_val_steps from None with 0 for Ganomaly                                                                                                                              engine.py:84
2024-06-25 16:39:51,464 - lightning.pytorch.utilities.rank_zero - INFO - Trainer already configured with model summary callbacks: [<class 'lightning.pytorch.callbacks.rich_model_summary.RichModelSummary'>]. Skipping setting a default `ModelSummary` callback.
[06/25/24 16:39:51] INFO     Trainer already configured with model summary callbacks: [<class 'lightning.pytorch.callbacks.rich_model_summary.RichModelSummary'>]. Skipping setting a default `ModelSummary`         rank_zero.py:63
                             callback.
2024-06-25 16:39:51,472 - lightning.pytorch.utilities.rank_zero - INFO - GPU available: True (cuda), used: True
                    INFO     GPU available: True (cuda), used: True                                                                                                                                                  rank_zero.py:63
2024-06-25 16:39:51,473 - lightning.pytorch.utilities.rank_zero - INFO - TPU available: False, using: 0 TPU cores
                    INFO     TPU available: False, using: 0 TPU cores                                                                                                                                                rank_zero.py:63
2024-06-25 16:39:51,473 - lightning.pytorch.utilities.rank_zero - INFO - IPU available: False, using: 0 IPUs
                    INFO     IPU available: False, using: 0 IPUs                                                                                                                                                     rank_zero.py:63
2024-06-25 16:39:51,474 - lightning.pytorch.utilities.rank_zero - INFO - HPU available: False, using: 0 HPUs
                    INFO     HPU available: False, using: 0 HPUs                                                                                                                                                     rank_zero.py:63
2024-06-25 16:39:51,475 - anomalib.models.components.base.anomaly_module - WARNING - No implementation of `configure_transforms` was provided in the Lightning model. Using default transforms from the base class. This may not be suitable for your use case. Please override `configure_transforms` in your model.
                    WARNING  No implementation of `configure_transforms` was provided in the Lightning model. Using default transforms from the base class. This may not be suitable for your use case. Please anomaly_module.py:235
                             override `configure_transforms` in your model.
2024-06-25 16:39:51,476 - anomalib.data.image.btech - INFO - Found the dataset.
                    INFO     Found the dataset.                                                                                                                                                                         btech.py:351
2024-06-25 16:39:51,514 - lightning.pytorch.utilities.rank_zero - INFO - You are using a CUDA device ('NVIDIA GeForce RTX 3090') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' | 'high')` which will trade-off precision for performance. For more details, read https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
                    INFO     You are using a CUDA device ('NVIDIA GeForce RTX 3090') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' | 'high')` which   rank_zero.py:63
                             will trade-off precision for performance. For more details, read https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
2024-06-25 16:39:52,727 - lightning.pytorch.accelerators.cuda - INFO - LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
[06/25/24 16:39:52] INFO     LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]                                                                                                                                                    cuda.py:58
┏━━━┳━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┓
┃   ┃ Name                  ┃ Type                     ┃ Params ┃
┡━━━╇━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━┩
│ 0 │ generator_loss        │ GeneratorLoss            │      0 │
│ 1 │ discriminator_loss    │ DiscriminatorLoss        │      0 │
│ 2 │ _transform            │ Compose                  │      0 │
│ 3 │ normalization_metrics │ MinMax                   │      0 │
│ 4 │ image_threshold       │ F1AdaptiveThreshold      │      0 │
│ 5 │ pixel_threshold       │ F1AdaptiveThreshold      │      0 │
│ 6 │ image_metrics         │ AnomalibMetricCollection │      0 │
│ 7 │ pixel_metrics         │ AnomalibMetricCollection │      0 │
│ 8 │ model                 │ GanomalyModel            │  188 M │
└───┴───────────────────────┴──────────────────────────┴────────┘
Trainable params: 188 M
Non-trainable params: 0
Total params: 188 M
Total estimated model params size (MB): 754
/home/jinyao/anaconda3/envs/IADBE/lib/python3.10/site-packages/lightning/pytorch/core/module.py:494: You called `self.log('generator_loss', ..., logger=True)` but have no logger configured. You can enable one by doing
`Trainer(logger=ALogger(...))`
/home/jinyao/anaconda3/envs/IADBE/lib/python3.10/site-packages/lightning/pytorch/core/module.py:494: You called `self.log('discriminator_loss', ..., logger=True)` but have no logger configured. You can enable one by doing
`Trainer(logger=ALogger(...))`
Epoch 7/99 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 32/32 0:00:07 • 0:00:00 4.85it/s generator_loss_step: 35.912 discriminator_loss_step: 50.000 image_AUROC: 0.341 image_PRO: 1.000 generator_loss_epoch: 36.363
                                                                                     discriminator_loss_epoch: 50.000
[06/25/24 16:41:38] INFO     Training took 105.83 seconds                                                                                                                                                                timer.py:59
Epoch 7/99 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 32/32 0:00:07 • 0:00:00 4.85it/s generator_loss_step: 35.912 discriminator_loss_step: 50.000 image_AUROC: 0.420 image_PRO: 1.000 generator_loss_epoch: 36.237
                                                                                     discriminator_loss_epoch: 50.000
2024-06-25 16:41:38,568 - anomalib.data.image.btech - INFO - Found the dataset.
                    INFO     Found the dataset.                                                                                                                                                                         btech.py:351
2024-06-25 16:41:38,736 - lightning.pytorch.accelerators.cuda - INFO - LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
[06/25/24 16:41:38] INFO     LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]                                                                                                                                                    cuda.py:58
Testing ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 14/14 0:00:23 • 0:00:00 0.56it/s  2024-06-25 16:42:04,563 - anomalib.callbacks.timer - INFO - Testing took 25.296730518341064 seconds
Throughput (batch_size=32) : 17.433082891097673 FPS
[06/25/24 16:42:04] INFO     Testing took 25.296730518341064 seconds                                                                                                                                                    timer.py:109
                             Throughput (batch_size=32) : 17.433082891097673 FPS
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃        Test metric        ┃       DataLoader 0        ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│        image_AUROC        │    0.4196951389312744     │
│         image_PRO         │            1.0            │
└───────────────────────────┴───────────────────────────┘
Testing ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 14/14 0:00:23 • 0:00:00 0.56it/s




